examples/tutorial_server_alarms_conditions.c

```diff
@@ -0,0 +1,508 @@
+        return;
+    }
+
+    UA_UInt16 severityValue = 100;
+    retval = UA_Server_setConditionField(server, &conditionInstance_1,
+                                         &severityValue, UA_TYPES_UINT16, &severityField);
+    if(retval != UA_STATUSCODE_GOOD) {
+        UA_LOG_ERROR(UA_Log_Stdout, UA_LOGCATEGORY_USERLAND,
+                     "Setting Severity Field failed. StatusCode %s", UA_StatusCode_name(retval));
+        return;
+    }
+
+    UA_LocalizedText messageValue = UA_LOCALIZEDTEXT("en", "Condition returned to normal state");
+    retval = UA_Server_setConditionField(server, &conditionInstance_1,
+                                         &messageValue, UA_TYPES_LOCALIZEDTEXT, &messageField);
+    if(retval != UA_STATUSCODE_GOOD) {
+        UA_LOG_ERROR(UA_Log_Stdout, UA_LOGCATEGORY_USERLAND,
+                     "Setting Message Field failed. StatusCode %s", UA_StatusCode_name(retval));
+        return;
+    }
+
+    UA_LocalizedText commentValue = UA_LOCALIZEDTEXT("en", "Normal State");
+    retval = UA_Server_setConditionField(server, &conditionInstance_1,
+                                         &commentValue, UA_TYPES_LOCALIZEDTEXT, &commentField);
+    if(retval != UA_STATUSCODE_GOOD) {
+        UA_LOG_ERROR(UA_Log_Stdout, UA_LOGCATEGORY_USERLAND,
+                     "Setting Comment Field failed. StatusCode %s", UA_StatusCode_name(retval));
+        return;
+    }
+
+    UA_Boolean retainValue = false;
+    retval = UA_Server_setConditionField(server, &conditionInstance_1,
+                                         &retainValue, UA_TYPES_BOOLEAN, &retainField);
+    if(retval != UA_STATUSCODE_GOOD) {
+        UA_LOG_ERROR(UA_Log_Stdout, UA_LOGCATEGORY_USERLAND,
+                     "Setting Retain Field failed. StatusCode %s", UA_StatusCode_name(retval));
+        return;
+    }
+
+    retval = UA_Server_triggerConditionEvent(server, conditionInstance_1, conditionSource, NULL);
+    if (retval != UA_STATUSCODE_GOOD) {
+     UA_LOG_WARNING(UA_Log_Stdout, UA_LOGCATEGORY_USERLAND,
+                    "Triggering condition event failed. StatusCode %s", UA_StatusCode_name(retval));
+     return;
+    }
+}
+
+static UA_StatusCode
+enteringEnabledStateCallback(UA_Server *server, const UA_NodeId *condition) {
+    UA_Boolean retain = UA_TRUE;
+
+    return UA_Server_writeObjectProperty_scalar(server, *condition, UA_QUALIFIEDNAME(0, "Retain"),
+                                                &retain, &UA_TYPES[UA_TYPES_BOOLEAN]);
+}
+
+/**
+ * this is user specific function which will be called upon acknowledging an alarm notification.
+ * In this example we will set the Alarm to Inactive state. The server is responsible of setting
+ * standard fields related to Acknowledge Method and triggering the alarm notification.
+ */
+static UA_StatusCode
+enteringAckedStateCallback(UA_Server *server, const UA_NodeId *condition) {
+    /* set ActiveState/Id to false*/
```

Comment:
this comment is "not carrying its own weight".

please compare "activeStateId = false;" with "set ActiveState/Id to false"